packages/coze-js/README.zh-CN.md

```
@@ -231,15 +231,15 @@ const client = new CozeAPI({
 
 [查看代理示例 →](../../examples/coze-js-node/src/proxy/)
 
-### 语音合成
+### 语音合成 SDK
+在线 Demo: https://wwww.coze.cn/open-platform/realtime/websocket#speech
```

markdownlint-cli2 (0.17.2)

Bare URL used
null

(MD034, no-bare-urls)